### DISCUSSION TITLE
Make OctoArcade Invaders Project Viewable to the public

### DISCUSSION THREAD

**Discussion by 0Grit** [community/community#8670] [Projects and Issues]
I'd like to refer to the OctoArcade Invaders project when looking at new features or advanced usecases of the new projects features. But I can't find it or any of the issues it tracks.

![image](https://user-images.githubusercontent.com/24318071/145603956-1299a5db-7d34-4736-8357-1b9e0daea214.png)

**Comment by zamli** [comment]
This is such a no brainer. I was shocked to find no link nowhere

**Comment by stoutgz** [comment]
You guys are still referencing this project in your latest examples. I also would like to refer to this project as a guide to what might be good practices in building out my project views.

**Comment by jarvisryan** [comment]
Please do share if you have it stored somewhere. We would love to use it as a base for our new Roadmap utilising the Projects feature.

**Comment by rickrickrickleee** [comment]
https://www.youtube.com/watch?v=7cn-dDpFktE I am currently watching this and found this link

**Comment by schneidergithub** [comment]
This is still hidden as of July 2026.
